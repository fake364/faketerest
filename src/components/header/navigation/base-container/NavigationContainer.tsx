import React from 'react';

type Props = {};

const NavigationContainer: React.FC<Props> = ({ children }) => {
	return <nav className="h-[80px] flex items-center justify-between pl-[20px] pr-[20px]">{children}</nav>;
};

export default NavigationContainer;
