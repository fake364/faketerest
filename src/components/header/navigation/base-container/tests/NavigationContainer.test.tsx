import React from 'react';
import { mount } from 'enzyme';
import NavigationContainer from '../NavigationContainer';
import { expect } from '@jest/globals';

describe('NavigationContainer', () => {
  it('should render container properly', () => {
    const component = mount(<NavigationContainer>Test</NavigationContainer>);
    expect(component).toMatchSnapshot();
  });
});
