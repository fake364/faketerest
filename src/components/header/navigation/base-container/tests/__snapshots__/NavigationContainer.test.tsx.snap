// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationContainer should render container properly 1`] = `
<NavigationContainer>
  <nav
    className="h-[80px] flex items-center justify-between pl-[20px] pr-[20px]"
  >
    Test
  </nav>
</NavigationContainer>
`;
